_source/api/logzio-public-api.yml

```diff
@@ -2701,6 +2705,50 @@ paths:
         - lang: cURL
           source: 'curl -X POST "https://api.logz.io/v1/shared-tokens" -H  "accept: application/json" -H  "content-type: application/json" -H  "X-API-TOKEN: <token>" -d "{ \"tokenName\": \"Support shared token\", \"filters\":[339]}"'
 
+  # ::::: API TOKENS
+  /v1/api-tokens/sub-account:
+    post: 
+      operationId: createSubaccountAPIToken
+      summary: Create a subaccount API token
+      description: >-
+        Creates a new API token for a subaccount.
+
+      tags:
+        - Manage API tokens
+      parameters: 
+        - in: body
+          name: body
+          required: true  
+          schema:
+            $ref: '#/definitions/CreateSubaccountAPITokenRequest'
```

Comment:
what's this schema?

Reply:
if this is a request object then it should be `CreateApiTokenRequest`

Reply:
I made this schema up, following the pattern of the shared token API. 

Changed object names: DONE